readthedocs/projects/tasks.py

```diff
@@ -61,6 +60,7 @@
 from readthedocs.doc_builder.loader import get_builder_class
 from readthedocs.doc_builder.python_environments import Conda, Virtualenv
 from readthedocs.projects.models import APIProject, Feature
+from readthedocs.search.signals import index_new_files, remove_indexed_files
```

Comment:
Those should be moved to another module, but another PR or after review, so is easy to see the diff :)